bin/addons/godot_whisper/capture_stream_to_text.gd

```diff
@@ -159,8 +156,15 @@ func _remove_special_characters(message: String, is_partial: bool):
 			var end_character := message.find(special_character["end"])
 			if end_character != -1:
 				message = message.substr(0, begin_character) + message.substr(end_character + 1)
-	
+
 	message = message.trim_suffix("{SPLIT}")
+	
+	var hallucinatory_character = [". you."]
```

Comment:
hm, this might work in English, but idk about other languages.

Reply:
Yes, because "you" often appear in my hallucinations, I have done some blocking. 
It is unreasonable to only use characters to block all hallucinatory characters, and the hallucinatory problem should be solved by whistler.cpp. So this is just a temporary solution